wp-content/plugins/pri-migration-helper/migration/taxonomy_filters/taxonomy-category.php

```diff
@@ -243,8 +243,12 @@ function pri_fgd2wp_pre_insert_taxonomy_term( $custom_field_values, $new_term_id
 	switch ( $custom_field_name ) {
 		case 'category-body':
 			if ( $custom_field_values && ! empty( $custom_field_values[0] ) ) {
-				$args['description'] = get_term_field( 'description', $new_term_id, 'category', 'raw' ) . ' ' . $custom_field_values[0]['field_body_value'];
-				wp_update_term( $new_term_id, 'category', $args );
+				$cat_description = get_term_field( 'description', $new_term_id, 'category', 'raw' );
+				if ( ! $cat_description && ! empty( $custom_field_values[0]['field_body_value'] ) ) {
```

Comment:
Should we do an empty check after trimming spaces on the `$cat_description` value, just in case it only contains space characters. I don't think `get_ferm_field` does that sanitization on raw context.

Reply:
Validation added using trim description.